src/Symfony/Component/Serializer/Normalizer/GetSetMethodNormalizer.php

```diff
@@ -47,7 +47,7 @@ public function denormalize($data, $class, $format = null, array $context = arra
         $normalizedData = $this->prepareForDenormalization($data);
 
         $reflectionClass = new \ReflectionClass($class);
-        $object = $this->instantiateObject($normalizedData, $class, $context, $reflectionClass, $allowedAttributes);
+        $object = $this->instantiateComplexObject($normalizedData, $class, $context, $reflectionClass, $allowedAttributes, $format);
```

Comment:
It would still be a problem:

```php
class MyCustomNormalizer extends AbstractObjectNormalizer
{
  protected instantiateObject(..., $allowedAttributes, $myParam = [])
  {
  }
```

And AFAIK it's a case where Symfony promises BC.